model_prices_and_context_window.json

```diff
@@ -1341,6 +1341,18 @@
         "mode": "chat",
         "source": "https://azuremarketplace.microsoft.com/en/marketplace/apps/metagenai.meta-llama-3-2-11b-vision-instruct-offer?tab=Overview"
     },
+    "azure_ai/Llama-3.3-70B-Instruct": {
+        "max_tokens": 128000,
```

Comment:
nit: max_tokens maps to max_output_tokens. not max_input_tokens @emerzon 

can you make this edit on a new pr